internal/operands/common-templates/resource.go

```diff
@@ -0,0 +1,129 @@
+package common_templates
+
+import (
+	"bytes"
+	templatev1 "github.com/openshift/api/template/v1"
+	"io"
+	"io/ioutil"
+	core "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/yaml"
+	"sync"
+)
+
+const (
+	goldenImagesNSname = "kubevirt-os-images"
+	version            = "v0.12.2"
+	bundleDir          = "../../../data/common-templates-bundle/"
```

Comment:
This path is relative to what? It may be good to start the operator with working directory at the root of the repo, or at the location of the binary.

Reply:
This is relative to the operand files

Reply:
This will probably not work on a deployed operator. The path should be relative to the binary or root of the repo, not the source files. The sources will not be present in the operator image.